public/index.html

```diff
@@ -2,39 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <!-- <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico"> -->
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <!--
-      manifest.json provides metadata used when your web app is added to the
-      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
-    -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>Software & Computer Engineering Society</title>
   </head>
   <body>
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
-    <!--
```

Comment:
I prefer keeping these comments here since it gives an idea of where react will render its components, but thats just me

Reply:
@ShivaniAsokumar could revert index.html or just get rid of the blank line on line 15

Reply:
I put the original comments back in the document, but for some reason, it keeps inserting a blank line between and the head and body tags even though I have prettier disabled. 

Here is what it looks like: 
![image](https://user-images.githubusercontent.com/62733107/91644077-9d12d700-e9ed-11ea-837d-c6e0844bddaf.png)

Is this okay?